NCS.DSS.Customer/ServiceBus/CustomerServiceBusClient.cs

```diff
@@ -0,0 +1,130 @@
+                await AutoSubscribeCustomer(customer);
+
+                _logger.LogInformation("Attempting to Send Service Bus Message for Customer with ID {CustomerID}", customer.CustomerId);
+                await serviceBusSender.SendMessageAsync(msg);
+                _logger.LogInformation("Service Bus Message for Customer with ID {CustomerID} has been sent successfully", customer.CustomerId);
+
+            }
+            catch (Exception ex)
+            {
+                _logger.LogInformation("Failed to Send Service Bus Message for Customer with ID {CustomerID}. Exception Raised with {Message}. {StackTrace}", customer.CustomerId, ex.Message,ex.StackTrace);
+                throw;
+            }            
+        }
+
+        public async Task SendPatchMessageAsync(CustomerPatch customerPatch, Guid customerId, string reqUrl)
+        {
+            try
+            {
+                _logger.LogInformation("Attempting to Create Sender for Service Bus Client");
+                var serviceBusSender = _serviceBusClient.CreateSender(QueueName);
+                _logger.LogInformation("Preparing Message for Service Bus");
+
+                var messageModel = new MessageModel
+                {
+                    TitleMessage = "Outcome record modification for {" + customerId + "} at " + DateTime.UtcNow,
+                    CustomerGuid = customerId,
+                    LastModifiedDate = customerPatch.LastModifiedDate,
+                    URL = reqUrl,
+                    IsNewCustomer = false,
+                    TouchpointId = customerPatch.LastModifiedTouchpointId,
+                    IsDigitalAccount = customerPatch.IsDigitalAccount,
+                    UpdateDigitalIdentity = customerPatch.UpdateDigitalIdentity,
+                    FirstName = customerPatch.GivenName,
+                    LastName = customerPatch.FamilyName,
+                    IdentityStoreId = customerPatch.IdentityStoreId,
+                    DeleteDigitalIdentity = customerPatch.DeleteDigitalIdentity
+                };
+
+                var msg = new ServiceBusMessage(Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(messageModel)))
+                {
+                    ContentType = "application/json",
+                    MessageId = customerId + " " + DateTime.UtcNow
+                };
+
+                _logger.LogInformation("Attempting to Send Service Bus Message for Customer with ID {CustomerID}", customerId);
+                await serviceBusSender.SendMessageAsync(msg);
+                _logger.LogInformation("Service Bus Message for Customer with ID {CustomerID} has been sent successfully", customerId);
+
+
+            }
+            catch (Exception ex)
+            {
+                _logger.LogInformation("Failed to Send Service Bus Message for Customer with ID {CustomerID}. Exception Raised with {Message}. {StackTrace}", customerId, ex.Message, ex.StackTrace);
```

Comment:
The stack trace is not required, the log level should be error and the log should pass the exception object:
`_logger.LogError(ex, "Failed to Send Service Bus Message for Customer with ID {CustomerID}. Exception Raised with {Message}", customerId, ex.Message);`